### ISSUE TITLE
Add autolabeling functionality to discord bot for new submissions

### ISSUE THREAD

**Issue by Conor12345** [AlternativeFFFF/Alt-F4#27]
Update github actions to automatically label submissions with the correct label if they fulfil submission criteria.

**Comment by DrMagPie** [comment]
This might be useful for auto labelling functionality https://github.com/marketplace/issue-label-bot

**Comment by LovelySanta** [comment]
Automatically labling is not the issue, the issue we have here is that we need to label PRs through the ALT F4 bot account in order to allo triggering another workflow when something is labeled in order to announce new submissions to discord. This is exactly what we tried with https://github.com/AlternativeFFFF/Alt-F4/blob/master/.github/workflows/disabled_workflows/Github_PR_Labeler.yml . The issue with on_pull_request is that it runs on the (forked) branch where the PR originates from instead of running it on the PR target (aka this repo). on_pull_request_target is the equivalent that recently has been added to resolve this issue, however finding a compatible action seems not so trivial.
